src/main/resources/static/api.yaml

```diff
@@ -372,6 +372,32 @@ paths:
             application/xml:
               schema:
                 $ref: '#/components/schemas/ConsentApprovalRepresentation'
+
+  /consents/{consent-id}:
+    get:
+      tags:
+        - consent
+      summary: List consent requests
```

Comment:
Get Consent? not list of consents

Reply:
no. it just gets a single CA. I am assuming that the CM notification will tell the HIU which all CA are created for a given request. Potentially different APIs can be created (does not exist yet)
- given a request Id, get the status - and possible links to the granted CAs. 
- GET consent-artefacts?request-id=xyz, where it will fetch list of consents. 
or instead of the above
- GET consent-artefacts?consent_id=XYZ1 & consent_id=XYZ2 ..

Reply:
I was talking about summary of the API.

Reply:
done